### PR TITLE
Templates shoes

### PR DESCRIPTION
**Pull request by gintama91**
### Description

Shoes app complimentary to #223

### Image(if needed, helps for a faster review)

Shoes app pic

image

usiing cli pic

image

### Checklist

- [ ] Run tests locally
- [x] Run linter(check for linter errors)

### CODE REVIEW

**Comment by gintama91** [comment]
this is branched out from templates branch in #223 so there are common files like `.erb` files , after #223  gets approval to merge i will resolve merge conflicts in this one

**Comment by gintama91** [comment]
ignore above comment, i close other pr as this has the code in tht pr too(ig so)

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```
Comment (noahgibbs) [inline .tool-versions:1]: Did you mean to check this in? This doesn't match the .ruby-version file.

File: README.md [hunk 2]
```diff
@@ -67,6 +67,10 @@ It's very early in the development process. If you'd like to help develop Scarpe
 
 We'd love the help!
 
+If you want to quickly add a feature, you can use the `ruby scarpegen.rb` command. This command will generate the necessary files for you, along with a simple template and a set of questions to guide you through the process. By following these steps, you will be good to go!
+
+By leveraging the `ruby scarpegen.rb` command and the provided resources, you can expedite the feature addition process and ensure a smoother development experience.
+
```
Comment (noahgibbs) [inline README.md:73]: Yay, adding to the README :-)

File: lib/scarpe/wv/button.rb [hunk 3]
```diff
@@ -24,9 +24,13 @@ def element
 
     def style
       styles = {}
-
+      styles["background-color"] = @color
```
Comment (noahgibbs) [inline lib/scarpe/wv/button.rb:27]: Yeah... At some point we really need to go through Shoes and figure out what properties/styles we're supporting.

In the mean time, this is fine.

File: .tool-versions (hunk 1 above)
Comment (gintama91) [reply .tool-versions:1]: ah, hm yeah it got in accidentally

Comment (gintama91) [reply .tool-versions]: removed

**Review by Schwad — APPROVED** [review]
Using shoes to build Scarpe. Now that's what I'm talking about!

**Pull request merged**